pkg/plugins/dir.go

```diff
@@ -30,3 +32,34 @@ func writeDirectory(dir schema.Directory, fs vfs.FS) error {
 
 	return fs.Chown(dir.Path, dir.Owner, dir.Group)
 }
+
+func writePath(dir schema.Directory, fs vfs.FS, topLevel bool) error {
```

Comment:
I'm a bit confused here. The goal is to create all the path, like `mkdir -p`? In this case you could replace this call with [os.MkdirAll](https://golang.org/pkg/os/#MkdirAll)

Reply:
Well I did this way mostly to keep using the `fs vfs.FS` and have it all file and directory writes through the same interface.  I also I assumed to apply the provided permissions to any created directory, which is a design choice. I thought it is more natural to apply the provided ownership and permissions to any created directory rather than let it go with some default ownership. That's why I did not dig that much into the idea of using `MkdirAll` and thought about simply iterating over the already existing method.